rest/data/interface.go

```diff
@@ -143,14 +143,14 @@ type Connector interface {
 	SetPatchActivated(string, string, bool) error
 
 	// GetAdminSettings/SetAdminSettings retrieves/sets the system-wide settings document
-	GetAdminSettings() (*admin.AdminSettings, error)
-	SetAdminSettings(*admin.AdminSettings, *user.DBUser) error
+	GetAdminSettings() (*evergreen.Settings, error)
+	SetAdminSettings(*evergreen.Settings, *user.DBUser) error
```

Comment:
the names of these methods seems potentially weird now.

Reply:
renamed